internal/commands/whoami/command.go

```diff
@@ -51,5 +53,24 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 	}
 
 	ui.Print(terminal.NewTextLog("Currently logged in user: %s (%s)", userDisplay, userSecret))
+
+	groups, groupsErr := clients.Atlas.Groups()
+	if groupsErr == nil {
+		headerProjectID := "Project ID"
```

Comment:
[question] should these just be consts in the file, rather than vars within the command handler?